lib/puppet/provider/netapp_security_login/cmode.rb

```diff
@@ -0,0 +1,99 @@
+require 'puppet/provider/netapp_cmode'
+
+Puppet::Type.type(:netapp_security_login).provide(:cmode, :parent => Puppet::Provider::NetappCmode) do
+  @doc = "Manage Netapp security logins"
+
+  confine :feature => :posix
+  defaultfor :feature => :posix
+
+  netapp_commands :securityloginlist    => {:api => 'security-login-get-iter', :iter => true, :result_element => 'attributes-list'}
+  netapp_commands :securitylogincreate  => 'security-login-create'
+  netapp_commands :securitylogindestroy => 'security-login-delete'
+  netapp_commands :securityloginmodify  => 'security-login-modify'
+  netapp_commands :securityloginlock    => 'security-login-lock'
+  netapp_commands :securityloginunlock  => 'security-login-unlock'
+
+  mk_resource_methods
+
+  def self.instances
+    securitylogins = []
+    results = securityloginlist() || []
+    results.each do |securitylogin|
+      application = securitylogin.child_get_string('application')
+      authentication_method = securitylogin.child_get_string('authentication-method')
+      username = securitylogin.child_get_string('user-name')
+      vserver = securitylogin.child_get_string('vserver')
+      securitylogin_hash = {
+        :name              => "#{application}:#{authentication_method}:#{username}:#{vserver}",
+        :password          => securitylogin.child_get_string('password'),
+        :comment           => securitylogin.child_get_string('comment'),
+        :role_name         => securitylogin.child_get_string('role-name'),
+        :snmpv3_login_info => securitylogin.child_get_string('snmpv3-login-info'),
```

Comment:
I don't see snmpv3-login-info in security-login-account-info either, but I see it can go to security-login-create... so I think that means we either remove it, or make it a parameter. I'd vote remove it.